sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/AggregateBenchmark.scala

```diff
@@ -366,6 +366,43 @@ class AggregateBenchmark extends BenchmarkBase {
      */
   }
 
+  ignore("Capacity for fast hash aggregate") {
```

Comment:
I don't see why we need to add benchmark, we just make the capacity configurable.